sky/dag.py

```diff
@@ -91,22 +90,24 @@ def remove(self, task: Union['task.Task', str]) -> None:
             task: The Task object or name of the task to remove.
 
         Raises:
-            ValueError: If the task is still being depended on by other tasks.
+            ValueError: If the task is still being used as a downstream task by
```

Comment:
```suggestion
            ValueError: If the task is still being used as a upstream task by
```
is this an upstream or downstream? we are getting the downstream of `task`, so `task` should be their upstream?

Reply:
Nice catch! You're absolutely right, and I totally missed that.